dev-haskell/stack/stack-2.7.3.ebuild

```diff
@@ -1,18 +1,22 @@
 # Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
+
+# ebuild generated by hackport 0.7
+#hackport: flags: -disable-git-info,-hide-dependency-versions,-integration-tests,-static,-supported-build
```

Comment:
This line can be modified to tell hackport to disable `developer-mode` in later updates:

```
#hackport: flags: -disable-git-info,-hide-dependency-versions,-integration-tests,-static,-supported-build,-developer-mode
```